packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -116,9 +119,10 @@ const NavigationActions = (props: Props) => {
                     label={<Translation id="TR_BACKENDS" />}
                     icon="BACKEND"
                     isMobileLayout={props.isMobileLayout}
+                    indicator={customBackends.length ? 'check' : undefined}
```

Comment:
This is useless, right? User does not see Custom backends icon when `customBackends.length === 0`

Reply:
Currently it's not useless as Custom backend item is always visible in mobile layout. The question is whether it should be, because in normal layout it's hidden.

Reply:
![Screenshot 2022-03-08 at 10 59 35](https://user-images.githubusercontent.com/33235762/157213410-a5d3c684-1d00-4819-b21e-ed3792a8c188.png)
Btw, delimiter is missing there.

Reply:
let's do it the same as on wider screens - hide the Backends item until there is some custom backend. I believe the only reason why it wasn't implemented in the first place was, that the info wasn't available in this component, but that is solved by the `useCustomBackends` hook

Reply:
https://github.com/trezor/trezor-suite/pull/5053/commits/e32aa17063b3f62b568712ecc2b2b4d326a135d6 - Backends item hidden in mobile layout as well
https://github.com/trezor/trezor-suite/pull/5053/commits/691ff7acd6cc28b76250e83c2075e0092e28e221 - Separators fixed

Reply:
![Screenshot 2022-03-22 at 9 39 22](https://user-images.githubusercontent.com/33235762/159440851-13ea7388-13d1-420e-9cff-1af4079475da.png)
There is `margin-right: 16px` on menu items which causes bigger gap on right side.

Otherwise it is okay and you can squash it.